genai_cookbook/nbs/4-evaluation-metrics.md

```diff
@@ -1,17 +1,17 @@
 ## Assessing performance: Metrics that Matter
 
-With an evaluation set, you are able to measure the performance of your RAG application across a number of different dimensions, including:
+With an evaluation set, you are able to measure the performance of your Agent application across a number of different dimensions, including:
 
-- **Retrieval quality**: Retrieval metrics assess how successfully your RAG application retrieves relevant supporting data. Precision and recall are two key retrieval metrics.
-- **Response quality**: Response quality metrics assess how well the RAG application responds to a user's request. Response metrics can measure, for instance, if the resulting answer is accurate per the ground-truth, how well-grounded the response was given the retrieved context (e.g., did the LLM hallucinate), or how safe the response was (e.g., no toxicity).
-- **System performance (cost & latency):**  Metrics capture the overall cost and performance of RAG applications. Overall latency and token consumption are examples of chain performance metrics.
+- **Retrieval quality**: Retrieval metrics assess how successfully your Agent application retrieves relevant supporting data. Precision and recall are two key retrieval metrics.
+- **Response quality**: Response quality metrics assess how well the Agent application responds to a user's request. Response metrics can measure, for instance, if the resulting answer is accurate per the ground-truth, how well-grounded the response was given the retrieved context (e.g., did the LLM hallucinate), or how safe the response was (e.g., no toxicity).
+- **System performance (cost & latency):**  Metrics capture the overall cost and performance of Agent applications. Overall latency and token consumption are examples of chain performance metrics.
 
-It is very important to collect both response and retrieval metrics. A RAG application can respond poorly in spite of retrieving the correct context; it can also provide good responses on the basis of faulty retrievals. Only by measuring both components can we accurately diagnose and address issues in the application.
+It is very important to collect both response and retrieval metrics. A Agent application can respond poorly in spite of retrieving the correct context; it can also provide good responses on the basis of faulty retrievals. Only by measuring both components can we accurately diagnose and address issues in the application.
```

Comment:
```suggestion
It is very important to collect both response and retrieval metrics. An Agent application can respond poorly in spite of retrieving the correct context; it can also provide good responses on the basis of faulty retrievals. Only by measuring both components can we accurately diagnose and address issues in the application.
```